cristin-commons/src/main/java/no/unit/nva/cristin/model/JsonPropertyNames.java

```diff
@@ -70,5 +70,6 @@ public class JsonPropertyNames {
     public static final String UNIT = "unit";
     public static final String USER = "user";
     public static final String LABELS = "labels";
+    public static final String NEAREST_PART_OF = "nearestPartOf";
```

Comment:
What is this property designed to express?

Reply:
Parent_unit object as opposed to list parent_units. When parsing the list into partOf from the list the object nearestPartOf might be redundant